homeassistant/components/conversation/__init__.py

```diff
@@ -78,49 +70,31 @@ def async_set_agent(hass: core.HomeAssistant, agent: AbstractConversationAgent):
     hass.http.register_view(ConversationProcessView())
     hass.http.register_view(ConversationHandleView())
     hass.components.websocket_api.async_register_command(websocket_process)
-    hass.components.websocket_api.async_register_command(websocket_get_agent_info)
+    hass.components.websocket_api.async_register_command(websocket__get_agent_info)
     hass.components.websocket_api.async_register_command(websocket_set_onboarding)
 
     return True
 
 
-async def process(hass: core.HomeAssistant, text: str, conversation_id: str):
-    """Process text and get intent."""
-    agent = await get_agent(hass)
-    return await agent.async_process(text, conversation_id)
-
-
-async def get_intent(hass: core.HomeAssistant, text: str, conversation_id: str):
-    """Process text and get intent."""
-    try:
-        intent_result = await process(hass, text, conversation_id)
-    except intent.IntentHandleError as err:
-        intent_result = intent.IntentResponse()
-        intent_result.async_set_speech(str(err))
-
-    if intent_result is None:
-        intent_result = intent.IntentResponse()
-        intent_result.async_set_speech("Sorry, I didn't understand that")
-
-    return intent_result
-
-
 @websocket_api.async_response
 @websocket_api.websocket_command(
     {"type": "conversation/process", "text": str, vol.Optional("conversation_id"): str}
 )
 async def websocket_process(hass, connection, msg):
     """Process text."""
     connection.send_result(
-        msg["id"], await get_intent(hass, msg["text"], msg.get("conversation_id"))
+        msg["id"],
+        await _async_converse(
+            hass, msg["text"], msg.get("conversation_id"), connection.context(msg)
+        ),
     )
 
 
 @websocket_api.async_response
 @websocket_api.websocket_command({"type": "conversation/agent/info"})
-async def websocket_get_agent_info(hass, connection, msg):
+async def websocket__get_agent_info(hass, connection, msg):
```

Comment:
What does the double underscore mean?

Reply:
a typo when I mass renamed the `get_agent` to `_get_agent` to indicate that it was private :)